### PR TITLE
Added "Edit this page" link to docs

### PR DESCRIPTION
**Pull request by daertommy**
With this PR readers can suggest changes to the docs with an easy to use WYSIWYG markdown editor.

[This](https://holocron.so/github/pr/neoforged/Documentation/main/editor) is how the editor looks like for this repo.

https://github.com/remorses/docusaurus-example/assets/31321188/43c8bc68-3668-4a25-bf1c-a70eceab7bcb

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -80,6 +83,7 @@ const config = {
             path: "5.x",
           },
         },
+
```
Comment (TheMCLoveMan) [inline docusaurus.config.js:86]: Remove empty line

**Review by TheMCLoveMan — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (TheMCLoveMan) [inline docusaurus.config.js:86]: Remove empty line

```suggestion
```

**Comment by ChampionAsh5357** [comment]
Apologies, but this tool is unnecessary for our purposes. To be able to edit a single or multiple pages, you could just press the `.` button to open up a codespace with Visual Studio Code, no setup required. Additionally, the editors would be familiar with the technical aspects of git and Markdown, using their own preference instead.

**Comment by daertommy** [comment]
Thank you for the feedback! Holocron has both Markdown and WYSIWYG editors in case you change idea

**Pull request closed without merging**